v22.1/api-support-policy.md

```diff
@@ -0,0 +1,109 @@
+| [`cockroach` commands](cockroach-commands.html)                         | Non-programmable | N/A                                                                                    |                                                                                                                                                              |
+| [DB Console](ui-overview.html)                                          | Non-programmable | N/A                                                                                    | For programmable access to the information present in this tool, use the [Cluster API](cluster-api.html).                                                    |
+| [Advanced Debug endpoints](ui-debug-pages.html)                         | Reserved         | N/A                                                                                    |                                                                                                                                                              |
+
+### {{ site.data.products.db }}
+
+These interfaces are available in {{ site.data.products.serverless-plan }} and {{ site.data.products.dedicated }}.
+
+| Interface                                     | Type             | Versioning                                                                                                                                  | Notes                                                                                                                                                                 |
+|-----------------------------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Cloud API](../cockroachcloud/cloud-api.html) | Programmable     | Versioned independently from CockroachDB. Major versions available via URI parameter. Fine-grained versioning available via request header. |                                                                                                                                                                       |
+| `ccloud` API                                  | Mixed            | Versioned independently from CockroachDB.                                                                                                   | Default output is non-programmable. If the `–json` argument is specified in the CLI, output is formatted in a stable JSON structure, following the versioning scheme. |
```

Comment:
There are two separate tools: the `ccloud` CLI and CockroachDB Cloud API (CCAPI). I think we could use two separate entries here. I would suggest something like this:

| Interface                                     | Type             | Versioning                                                                                                                                  | Notes                                                                                                                                                                 |
|-----------------------------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [`ccloud` CLI](https://www.cockroachlabs.com/docs/cockroachcloud/ccloud-get-started.html) | Mixed | Versioned independently from CockroachDB. | Default output is non-programmable. If the `–json` argument is specified in the CLI, output is formatted in a stable JSON structure, following the versioning scheme. |
| [CockroachDB Cloud API](https://www.cockroachlabs.com/docs/api/cloud/v1.html) | Programmable | Versioned independently from CockroachDB. | |